packages/tools/src/types/EventTypes.ts

```diff
@@ -20,6 +23,24 @@ type NormalizedMouseEventDetail = {
   element: HTMLDivElement;
 };
 
+/**
+ * The normalized mouse event detail
+ */
+type NormalizedTouchEventDetail = {
```

Comment:
This doesn't have anything different from the normalized mouse event detail - please re-use the existing type (you can rename it if you want, but only have one instance of it)